inst/shinyApps/noric/server.R

```diff
@@ -380,6 +380,13 @@ shinyServer(function(input, output, session) {
       fun = "subscriptionLocalMonthlyReps",
       paramNames = pn,
       paramValues = c("NORIC_local_monthly_stent", pv)
+    ),
+    Aktivitet = list(
+      synopsis = "M\u00E5nedlig oppsummering av aktiviteter siste \u00E5r",
+      fun = "reportProcessor",
+      paramNames = c("report", "outputType", "title", "orgId", "orgName", "userRole"),
```

Comment:
subreports <- list( 
Prosedyrer = ..., 
Stentbruk = ...)

if(is.nasjoantl = FALSE+ user.role= SC) {
subreports <- list(subreports, 
Aktivitet) }

Reply:
Her angis listen over mulige abonnement . 
Default er STENT + PROSEDYRER
Dersom lokal SC blir det STENT + PROSEDYRER + AKTIVITET